clang/lib/AST/ItaniumMangle.cpp

```diff
@@ -1699,6 +1704,33 @@ void CXXNameMangler::mangleTemplateParamDecl(const NamedDecl *Decl) {
   }
 }
 
+static void mangleUniqueNameLambda(CXXNameMangler &Mangler, SourceManager &SM,
+                                   raw_ostream &Out,
+                                   const CXXRecordDecl *Lambda) {
+  SourceLocation Loc = Lambda->getLocation();
+
+  PresumedLoc PLoc = SM.getPresumedLoc(Loc);
+  Mangler.mangleNumber(PLoc.getLine());
+  Out << "->";
+  Mangler.mangleNumber(PLoc.getColumn());
+
+  while (Loc.isMacroID()) {
```

Comment:
While I would love more comments in this PR in general, here it is even more obvious...